cas/service.c

```diff
@@ -1011,8 +1015,36 @@ static void cas_fom_failure(struct cas_fom *fom, int rc, bool ctg_op_fini)
 	m0_fom_phase_move(&fom->cf_fom, rc, M0_FOPH_FAILURE);
 }
 
+static int cas_dtm0_logrec_add (struct m0_fom *fom0, struct m0_dtm0_tx_desc *txd,
```

Comment:
s/static int cas_dtm0_logrec_add (struct m0_fom *fom0
/static int cas_dtm0_logrec_add(struct m0_fom *fom0

No space between the function name and the opening parenthesis.

Reply:
Done.